src/Entries/Collection.php

```diff
@@ -860,15 +860,9 @@ public function __toString()
 
     public function augmentedArrayData()
     {
-        $data = [
+        return [
             'title' => $this->title(),
             'handle' => $this->handle(),
         ];
-
-        if (! Statamic::isApiRoute() && ! Statamic::isCpRoute()) {
-            $data['mount'] = $this->mount();
```

Comment:
@jasonvarga Just came across this one again... do you think any of these options would work?

1. Instead of returning the actual mount, is there any way we could just return the shallowly augmented entry? Or would that not be useful enough?
2. In the same vein, could we have it augment properly but have it shalowly augment any relationship fields? Similar to [what I'm doing here](https://github.com/duncanmcclean/simple-commerce/blob/7.x/src/Http/Controllers/CartController.php#L27)?
3. Could we make `{{ mount }}` into a tag? Would that be useful enough?

Maybe (*probably*) those are all terrible answers.